packages/tree/src/CirclePacking.ts

```diff
@@ -58,7 +58,9 @@ export class CirclePacking extends SVGWidget {
         this.circle = this.svg.selectAll("circle").data(root.descendants())
             .enter().append("circle")
             .attr("class", function (d) { return d.parent ? d.children ? "node" : "node leaf" : "node root"; })
-            .style("fill", function (d) { return context._palette(d.data.label); })
+            .style("fill", function (d) {
+                return d.depth <= context.paletteDepthLevel() ? context._palette(d.data.label) : d3Rgb(context._palette(d.parent.label)).darker(1);
```

Comment:
I suspect this isn't quite doing what you want it to do?  
Visually are you trying to ensure that every circle is simply a darker color of its parent up to level 1 (where level 0 is the outer circle)?


I would have thought that after depth > 1 every nested child should be just a darker version of the top mosp

Reply:
Yes that is/was the goal. I see what you mean.